### ISSUE TITLE
EXO 34:16 ULB

### ISSUE THREAD

**Issue by SusanQuigley** [Door43/ulb-en#206]
EXO 34:16 ULB  You even take some of their daughters for your sons, and their daughters prostitute themselves to their gods, and they make your sons prostitute themselves to their gods. 

This reads in English as if God is asserting that these are things that his people were currently doing. But this was a warning that they might do these things if they make a covenant with the inhabitants of the land. Could "Otherwise" be added to the ULB, and could it be in the future tense?  (ESV uses "lest" at the beginning of 15, and 16 is part of that.)

**Otherwise**, you **will** even take some of their daughters for your sons, and their daughters **will** prostitute themselves to their gods, and they **will** make your sons prostitute themselves to their gods. 

If "for" or "Otherwise" is added to 34:15, it might be better to change 34:16 to something like this.

**And** you **will** even take some of their daughters for your sons, and their daughters **will** prostitute themselves to their gods, and they **will** make your sons prostitute themselves to their gods. 

(The word "even" probably is not needed in the first clause.)

**Comment by TomWarren** [comment]
The ULB of EXO 34:15-17 has been changed to avoid the confusion if time and causality between the idolatry of those in the land and the possible consequences of making a covenant with them.
 It now reads: 
\s5
\v 15 And so be careful not to make a covenant with the inhabitants of the land, for they prostitute themselves to their gods and they sacrifice to their gods, and someone invites you and you eat some of his sacrifice. 
\v 16 Then you will even take some of their daughters for your sons, and their daughters will prostitute themselves to their gods, and they will make your sons prostitute themselves to their gods.
\v 17 Do not make for yourselves gods of molten metal.

Thanks.

**Comment by pjoakes** [comment]
The second part of v. 15 (in red below) doesn’t make sense.

Perry Oakes
perry_oakes@wycliffeassociates.org<mailto:perry_oakes@wycliffeassociates.org>



On Aug 22, 2016, at 1:41 PM, Tom Warren <notifications@github.com<mailto:notifications@github.com>> wrote:


The ULB of EXO 34:15-17 has been changed to avoid the confusion if time and causality between the idolatry of those in the land and the possible consequences of making a covenant with them.
It now reads:
\s5
\v 15 And so be careful not to make a covenant with the inhabitants of the land, for they prostitute themselves to their gods and they sacrifice to their gods, and someone invites you and you eat some of his sacrifice.
\v 16 Then you will even take some of their daughters for your sons, and their daughters will prostitute themselves to their gods, and they will make your sons prostitute themselves to their gods.
\v 17 Do not make for yourselves gods of molten metal.

Thanks.

**Comment by TomWarren** [comment]
The rendering seemed to parallel the NIV and other modern readings. The repetition of "their gods" in following the Heb text. It is a bit wordy, but the Eng seemed to support it. Here is a slightly groomed version.  TW

The ULB of EXO 34:15-16 changed  (in vs 15) to read: 
\v 15 And so be careful not to make a covenant with the inhabitants of the land, for they prostitute themselves to their gods, and they sacrifice to their gods. Then one of them will invite you and you will eat some of his sacrifice, 
\v 16 and then you will even take some of his daughters for your sons, and his daughters will prostitute themselves to their gods, and they will make your sons prostitute themselves to their gods.
\v 17 Do not make for yourselves gods of molten metal.
